docs/feature_haptic_feedback.md

```diff
@@ -162,4 +162,23 @@ This will set what sequence HPT_RST will set as the active mode. If not defined,
 
 ### DRV2605L Continuous Haptic Mode
 
-This mode sets continuous haptic feedback with the option to increase or decrease strength. 
+This mode sets continuous haptic feedback with the option to increase or decrease strength.
+
+## Haptic Key Exclusion
+
+With the entry of `#define HAPTIC_EXCLUSION_KEYS` in config.h, exclusion keys can be enabled.
+By default they contain modifier and layer keys, which won't activate the haptic feedback.
+
+### Additional Haptic Key Exclusions
+
+In config.h keys can be defined in a comma-separated list which will excluded from activating a haptic feedback. The list has to be follow the form of:
+```
+#define HAPTIC_EXCLUSION_KEY_ADDITIONAL { KC_A, KC_B, KC_C }
+```
+
+### Default Key Exclusion De-activation
+
+In case it is desired to exclude only the self-defined and comma-separated list (without having the modifier and layer keys excluded), it can be achieved by turning off the default set in config.h:
+```
+#define HAPTIC_EXCLUSION_KEY_DEFAULT(keycode) 0
```

Comment:
I would really prefer that this be a function, rather than a define.  That would allow better customizability, and would be more in line with other behavior (tap-hold config). 

Could have a default function that has configuration to exclude keys and such. A good example of what I mean would be the auto shift per key stuff.
https://github.com/qmk/qmk_firmware/blob/6620b6c5155a9139b210d1fbec526eb3ebac5de8/quantum/process_keycode/process_auto_shift.c#L230-L246